spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/MaxPooling.scala

```diff
@@ -36,6 +37,8 @@ class MaxPooling(
   @transient private var paddingBR: Array[Int] = _
   @transient private var fwdPD: Long = _
 
+  var isOnlyForInference = false
```

Comment:
private?

Reply:
Delete this variable now to prevent bugs like `SpatialBatchNormalization`.